### PR TITLE
Introduction_to_Html_Audio

### LINKED ISSUE #225: Front End : Introduction to HTML
I would like to work on this issue in the form of Audio. 
Please assign it to me.
Regards.

### PR DESCRIPTION
**Pull request by I-ArchanaDash**
<hr>

## Description 📜
This Audio gives an Introduction to HTML 
Fixes  #225

<hr>

## Type of change 📝

- [x] Audio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Web_Development/FrontEnd/Introduction_to_Html/Audio.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Introduction to HTML Audio
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/Introduction_to_Html/Audio.md:1]: Enclose "Audio" in parenthesis.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Remove the second file.

File: Web_Development/FrontEnd/Introduction_to_Html/Audio.md (hunk 1 above)
Comment (I-ArchanaDash) [reply Web_Development/FrontEnd/Introduction_to_Html/Audio.md]: Done

**Comment by I-ArchanaDash** [comment]
> Remove the second file.

Do you mean the image file ?

**Comment by kashishahuja2002** [comment]
> Do you mean the image file ?

Yes merge it in the first file only.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
I can still see 2 files, remove Web_Development/FrontEnd/Introduction_to_Html/Audio/img.png

**Review by kashishahuja2002 — APPROVED** [review]
Good to go. @Saviour1001

**Review by praveenscience — APPROVED** [review]
👍

**Pull request merged**